scriptlets/test-executions-rerunner/tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
+minversion = 4
+envlist = py38
```

Comment:
Seems to rely on having py38 on your system? Is there a way to have tox ensure this exists? If not, then maybe it's better to have it use the system installed python and use a github workflow to ensure we have the right version? Otherwise just running tox here doesn't work